catroid/src/org/catrobat/catroid/formulaeditor/SensorHandler.java

```diff
@@ -177,10 +197,20 @@ public void onAccuracyChanged(Sensor arg0, int arg1) {
 	@Override
 	public void onSensorChanged(SensorEvent event) {
 		switch (event.sensor.getType()) {
+			case Sensor.TYPE_ACCELEROMETER:
+				accelerationXYZ = event.values.clone();
+				double norm_Of_g = Math.sqrt(accelerationXYZ[0] * accelerationXYZ[0]
```

Comment:
@camelCase, of course.
Shall I close the pr, change it to camelCase, squash the commits and open another pr?
Or is there a better wf?

@code: how shall I reference a partial code snippet from stackoverflow or developer.android.com?

Thanks for info.